src/geo.c

```diff
@@ -454,51 +481,61 @@ void geoaddCommand(client *c) {
 
 #define RADIUS_COORDS (1<<0)    /* Search around coordinates. */
 #define RADIUS_MEMBER (1<<1)    /* Search around member. */
-#define RADIUS_NOSTORE (1<<2)   /* Do not acceot STORE/STOREDIST option. */
+#define RADIUS_NOSTORE (1<<2)   /* Do not accept STORE/STOREDIST option. */
+#define GEOSEARCH (1<<3)        /* GEOSEARCH command variant (different arguments supported) */
+#define GEOSEARCH_NOSTORE (1<<4)   /* GEOSEARCH don't accept STORE/STOREDIST option,
```

Comment:
i would have negated this flag (i.e. call it `GEOSEARCHSTORE`)

Reply:
> i would have negated this flag (i.e. call it GEOSEARCHSTORE)

I will update to `GEOSEARCHSTORE`

> did you duplicate some of the GEORADIUS tests and changed them to use GEOSEARCH?

Yes, some tcl tests are done like this

> search for points that are outside a circle but inside a big that has similar "radius"

yes, see `test {GEOSEARCH vs GEORADIUS}`

> trying to create odd box shapes (non square ones)

Not yet, I will add some tests.